pq-crypto/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+file(GLOB GENERAL_HEADERS
+    "*.h"
+)
+
+file(GLOB BIKE_HEADERS
+    "bike/*.h"
+)
+
+file(GLOB SIKE_HEADERS
+    "sike/*.h"
+)
+
+file(GLOB GENERAL_SRC
+    "*.c"
+)
+
+file(GLOB BIKE_SRC
+    "bike/*.c"
+)
+
+file(GLOB SIKE_SRC
+    "sike/fp_generic.c"
```

Comment:
Why not `sike/*.c`?

Reply:
Yeah I'll update that, and remove the BIKE stuff since that will get added in the BIKE pull request.

Reply:
Okay turns out there was some additional complexity with how SIKE expects to be built. I also took the compile options from the top level CMakeLists.txt and updated the PQ crypto one. I need to find a way to share those between CMakeLists like the nice common s2n.mk.